rotary_utils/repair.py

```diff
@@ -906,16 +833,29 @@ def run_end_repair(long_read_filepath: str, assembly_fasta_filepath: str, assemb
                 f'A summary of repair work is saved at {end_repair_status_filepath}.')
 
 
-def parse_cli():
+def parse_cli(subparsers=None):
```

Comment:
For now, I keep the CLI parser in `repair.py` and then call this in `rotary_utils.py` as a subparser. Let me know if this is odd and the CLI parser for `repair.py` should be fully moved into `rotary_utils.py`.

Reply:
@jmtsuji I think this should work.

Reply:
Sounds good -- thanks.